_studio/mfx_lib/shared/src/mfx_mpeg2_encode_vaapi.cpp

```diff
@@ -1696,7 +1696,9 @@ mfxStatus VAAPIEncoder::FillBSBuffer(mfxU32 nFeedback,mfxU32 nBitstream, mfxBits
     //------------------------------------------
     // (1) mapping feedbackNumber -> surface & bs
     bool isFound = false;
+#if !VA_CHECK_VERSION(1,9,0)
```

Comment:
I added this because waitSurface variable is used only in `#else` branch in the following piece of code:
https://github.com/Intel-Media-SDK/MediaSDK/blob/5e089305f1b95b51cee14fb976685e9c643f170c/_studio/mfx_lib/shared/src/mfx_mpeg2_encode_vaapi.cpp#L1742-L1760